src/h5web/vis-packs/core/containers/MatrixVisContainer.tsx

```diff
@@ -23,7 +22,8 @@ function MatrixVisContainer(props: VisContainerProps) {
   const axesCount = Math.min(dims.length, 2);
   const [dimMapping, setDimMapping] = useDimMappingState(dims, axesCount);
 
-  const { valuesStore } = useContext(ProviderContext);
+  const formatter = getFormatter(entity);
```

Comment:
New `getFormatter` utility, which returns a formatter for `number | string | boolean | H5WebComplex`. `MappedMatrixVis` and `MatrixVis` no longer have generics.